test/e2e/tests/confirmations/transactions/metrics.spec.ts

```diff
@@ -0,0 +1,212 @@
+/* eslint-disable @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires */
+import { strict as assert } from 'assert';
+import { MockedEndpoint, MockttpServer } from 'mockttp';
+import {
+  AnonymousTransactionMetaMetricsEvent,
+  TransactionMetaMetricsEvent,
+} from '../../../../../shared/constants/transaction';
+import { Driver } from '../../../webdriver/driver';
+import {
+  confirmContractDeploymentTransaction,
+  confirmDepositTransaction,
+  createContractDeploymentTransaction,
+  createDepositTransaction,
+} from './shared';
+
+const {
+  defaultGanacheOptionsForType2Transactions,
+  openDapp,
+  unlockWallet,
+  WINDOW_TITLES,
+  withFixtures,
+  getEventPayloads,
+} = require('../../../helpers');
+const FixtureBuilder = require('../../../fixture-builder');
+
+describe('Metrics', function () {
```

Comment:
Since integration tests are now merged, would this be a great use case for one, rather than adding more E2E tests for functionality that won't directly impact the user?

Reply:
I agree this is a use case, but I would prefer to avoid deleting working e2e tests in this PR